go.mod

```diff
@@ -15,9 +15,9 @@ require (
 	github.com/go-chi/chi/v5 v5.0.7
 	github.com/go-chi/render v1.0.1
 	github.com/go-gota/gota v0.12.0
-	github.com/go-testfixtures/testfixtures/v3 v3.5.0
+	github.com/go-testfixtures/testfixtures/v3 v3.7.0
 	github.com/golang-migrate/migrate/v4 v4.14.1
-	github.com/golang/protobuf v1.5.3 // indirect
+	github.com/golang/protobuf v1.5.4 // indirect
```

Comment:
Can we close out this PR: https://github.com/CMSgov/bcda-app/pull/923?

Reply:
that PR references a different package (weirdly), so we can check in to see if that PR can be merged.